ui-events/src/pointer/buttons.rs

```diff
@@ -53,19 +87,19 @@ impl PointerButtons {
     /// Add the `button` to the set.
     #[inline]
     pub fn insert(&mut self, button: PointerButton) {
```

Comment:
If we keep `None`, we should document either here or on the `None` variant that this is a no-op.